requirements.txt

```diff
@@ -5,55 +5,87 @@ aiosignal==1.3.2
 annotated-types==0.7.0
 anyio==4.7.0
 attrs==24.3.0
+cachetools==5.5.0
 certifi==2024.12.14
 cffi==1.17.1
 charset-normalizer==3.4.0
 cryptography==44.0.0
 dataclasses-json==0.6.7
 Deprecated==1.2.15
 distro==1.9.0
+eval_type_backport==0.2.0
+filelock==3.16.1
+filetype==1.2.0
 frozenlist==1.5.0
+fsspec==2024.10.0
+google-ai-generativelanguage==0.6.10
```

Comment:
🌐 Confirm that `google-ai-generativelanguage==0.6.10` is necessary for your application. If not, consider removing it.